gridmenuview/src/main/res/layout/card_menu_item.xml

```diff
@@ -2,7 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/square_layout"
     android:orientation="vertical"
-    android:layout_width="wrap_content"
+    android:layout_width="match_parent"
```

Comment:
Sorry, why this is changed?

Reply:
because if the size of every icon is different, then the size of the displayed icon will be different. so i change it to match_parent, to match all the icon to same size.

match_parent
![image](https://user-images.githubusercontent.com/45010500/84585308-d8fbbe80-ae38-11ea-9fb9-b2d8ff03377b.png)

wrap_content
![image](https://user-images.githubusercontent.com/45010500/84585315-ed3fbb80-ae38-11ea-825f-dbc96d624858.png)

Reply:
I see. Cool 👍